<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-R8SHTIK" failures="1" tests="5" name="com.test.HomePageTestCases" time="6.214" errors="0" timestamp="2023-01-29T12:44:11 IST" skipped="2">
  <testcase classname="com.test.HomePageTestCases" name="clickOnSignUpButton" time="4.255"/>
  <system-out/>
  <testcase classname="com.test.HomePageTestCases" name="clickOnLoginButton" time="1.949"/>
  <system-out/>
  <testcase classname="com.test.HomePageTestCases" name="userNameVerification" time="0.010">
    <failure message="Login Error! User Name Invalid! expected [Jabajaba] but found [Jabajaba123]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Login Error! User Name Invalid! expected [Jabajaba] but found [Jabajaba123]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.test.HomePageTestCases.userNameVerification(HomePageTestCases.java:67)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- userNameVerification -->
  <system-out/>
  <testcase classname="com.test.HomePageTestCases" name="clickOnContactButton">
    <skipped/>
  </testcase> <!-- clickOnContactButton -->
  <system-out/>
  <testcase classname="com.test.HomePageTestCases" name="clickOnAboutUs">
    <skipped/>
  </testcase> <!-- clickOnAboutUs -->
  <system-out/>
</testsuite> <!-- com.test.HomePageTestCases -->
